Tasks/AzurePowerShell/AzurePowerShell.ps1

```diff
@@ -19,11 +21,26 @@ if ($scriptArguments -match '[\r\n]') {
     throw (Get-VstsLocString -Key InvalidScriptArguments0 -ArgumentList $scriptArguments)
 }
 
+if($targetAzurePs -eq "OtherVersion") {
```

Comment:
will be better if we can make "OtherVersion""LatestVersion" as constant variables